modules/GKB/Release/Steps/UpdateDOIs.pm

```diff
@@ -25,17 +30,19 @@ override 'run_commands' => sub {
     my ($self, $gkbdir) = @_;
 
     my $host = $self->host;
-
     $self->cmd("Backing up databases $db and $gkcentral",
         [
             ["mysqldump -u$user -p$pass -h$host --lock-tables=FALSE $db > $db.dump"],
             ["mysqldump -u$user -p$pass -h$gkcentral_host --lock-tables=FALSE $gkcentral > $gkcentral.dump"]
         ]
     );
-    
-    my $live_run = $self->user_input->{'live_run'}->{'response'} =~ /^y/i ? '-live_run' : '';
-    my @args = ("-user", $user, "-pass", $pass, "-release_db", $db, "-release_db_host", $host, "-curator_db", $gkcentral, "-curator_db_host", $gkcentral_host, $live_run);
-    $self->cmd("Running script to update DOIs for $db and $gkcentral",[["perl update_dois.pl @args > update_dois.out 2>> update_dois.err"]]);
+
+    if ($self->user_input->{'props'}->{'response'} !~ /^y/i) {
+      die "Please update the databaseTR, and authorID fields in update_dois/config.properties before running.\n";
```

Comment:
What is "databaseTR"? Are there some simple instructions you could put into the script as comments?

Reply:
TR stands for Test Reactome, while GK indicates the GK_Central database. I'll add those comments

Reply:
Ok, the "TR" short-form wasn't clear to me.